examples/translate-features.html

```diff
@@ -44,8 +46,8 @@ <h4 id="title">Drag features example</h4>
     </div>
 
     <script src="../resources/jquery.min.js" type="text/javascript"></script>
+    <script src="loader.js?id=translate-features" type="text/javascript"></script>
```

Comment:
Minor, but tt makes sense to me to add the `loader.js` script tag after the `example-behaviour.js` script tag.